app/models/compute_resources/foreman/model/libvirt.rb

```diff
@@ -27,6 +27,12 @@ def provided_attributes
       super.merge({:mac => :mac})
     end
 
+    def host_compute_attrs(host)
+      super.tap do |attrs|
+        attrs[:arch] = host.architecture.try(:name) || "x86_64"
```

Comment:
Why do we hardcode this? Isn't architecture a required relation in our model?

Reply:
[only for managed hosts](https://github.com/theforeman/foreman/blob/develop/app/models/host/managed.rb#L319), if you import the VM from compute resource, it does not have to have architecture.